Shared/SwiftfinStore/SwiftfinStoreDefaults.swift

```diff
@@ -41,6 +41,7 @@ extension Defaults.Keys {
 	// Customize settings
 	static let showPosterLabels = Key<Bool>("showPosterLabels", default: true, suite: SwiftfinStore.Defaults.generalSuite)
 	static let showCastAndCrew = Key<Bool>("showCastAndCrew", default: true, suite: SwiftfinStore.Defaults.generalSuite)
+	static let showFlattenView = Key<Bool>("showFlattenView", default: true, suite: SwiftfinStore.Defaults.generalSuite)
```

Comment:
Let's change the string in English to `Flatten Library Items`, as we aren't toggling to show/hide something explicit like the poster labels.